test/cli/commands_test.py

```diff
@@ -1109,6 +1109,28 @@ def test_cli_get_default_config():
     assert config.get("verbose") == 2
 
 
+def test_cli_existing_redshift_bug():
+    """Documents existing errors with linting super data types in Redshift.
+
+    Original Issue: https://github.com/sqlfluff/sqlfluff/issues/1672 .
+    """
```

Comment:
This feels weird. We're adding a test to provide how wrong we are.

I think these belong as GitHub issues, rather than items in the code.

Reply:
Agreed. We can address the existing bugs in #2567 . I'll remove this test from the codebase.

Reply:
Done!